code/core/src/core-server/presets/common-preset.ts

```diff
@@ -104,8 +104,32 @@ export const favicon = async (
 
 export const babel = async (_: unknown, options: Options) => {
   const { presets } = options;
-
-  return presets.apply('babelDefault', {}, options);
+  const babelDefault = await presets.apply('babelDefault', {}, options);
+  return {
+    ...babelDefault,
+    // This override makes sure that we will never transpile babel further down then the browsers that storybook supports.
+    // This is needed to support the mount property of the context described here:
+    // https://storybook.js.org/docs/writing-tests/interaction-testing#run-code-before-each-test
+    overrides: [
+      {
+        include: /(story|stories)\.[cm]?[jt]sx?$/,
+        presets: [
+          [
+            '@babel/preset-env',
+            {
+              bugfixes: true,
+              targets: {
```

Comment:
Since we are setting the target, do you think it might be a nice performance optimisation to set `ignoreBrowserslistConfig` to true?

Reply:
The targets option always takes prio over the browserlist option. Double checked it.